crates/sui-bridge-indexer/src/eth_bridge_indexer.rs

```diff
@@ -90,38 +90,49 @@ impl Datasource<RawEthData, PgBridgePersistent, ProcessedTxnData> for EthSubscri
             let mut cached_blocks: HashMap<u64, Block<H256>> = HashMap::new();
 
             let mut stream = eth_ws_client.subscribe_logs(&filter).await?;
-            while let Some(_log) = stream.next().await {
-                let log = RawEthLog {
-                    block_number: _log
+            while let Some(log) = stream.next().await {
+                let raw_log = RawEthLog {
+                    block_number: log
                         .block_number
                         .ok_or(BridgeError::ProviderError(
                             "Provider returns log without block_number".into(),
                         ))
                         .unwrap()
                         .as_u64(),
-                    tx_hash: _log
+                    tx_hash: log
                         .transaction_hash
                         .ok_or(BridgeError::ProviderError(
                             "Provider returns log without transaction_hash".into(),
                         ))
                         .unwrap(),
-                    log: _log,
+                    log,
                 };
 
-                let block_number = log.block_number();
+                let block_number = raw_log.block_number();
 
                 let block = if let Some(cached_block) = cached_blocks.get(&block_number) {
                     cached_block.clone()
                 } else {
-                    let block = eth_ws_client.get_block(block_number).await?.unwrap();
+                    let block = retry_with_max_elapsed_time!(
```

Comment:
lets emit better error log instead of unwrap 

```
            let Ok(Ok(block)) =
                retry_with_max_elapsed_time!(.....)
            else {
                panic!("");
            };
```